libs/external-db-dynamodb/src/dynamo_schema_provider.ts

```diff
@@ -129,7 +151,7 @@ export default class SchemaProvider implements ISchemaProvider {
         const { Item } = await this.docClient
                                    .get(dynamoRequests.getCollectionFromSystemTableExpression(collectionName))
 
-        if (!Item && !toReturn ) throw new CollectionDoesNotExists('Collection does not exists')
+        if (!Item && !toReturn ) throw new CollectionDoesNotExists('Collection does not exists', collectionName)
         return Item
```

Comment:
maybe discribe the Item type, so you will not have to discribe the type in all the places that use this function. 
```suggestion
        return Item as { tableName: string, fields: {name: string, type: string, subtype: string}[] }
```